src/LobbyServer.hs

```diff
@@ -293,11 +294,41 @@ findClient clientName = find ((clientName ==).name)
 messageClients :: LobbyMessage -> [ClientEntry] -> IO ()
 messageClients m = mapM_ (\c -> CC.writeChan (lobbyChannel c) m)
 
+-- |Changes the maximum number of players for a game
+-- Requires that the player is the last in the player list (i.e. the owner)
+changeMaxNumberOfPlayers :: Server GamesList -> Int -> Server ()
+changeMaxNumberOfPlayers remoteGames newMax = do
+  mVarGames <- remoteGames
+  bool <- isOwnerOfGame remoteGames
```

Comment:
Don't name the variable `bool`. Name it so it explains what it do e.g. `clientOwnsGame` or something along those lines.